.travis.yml

```diff
@@ -71,3 +71,26 @@ jobs:
         - cmake --build . -- -j2
         - ctest -j2
 
+    - stage: compile and test
+      os: osx
+      env:
+        - TEST="Xcode Check"
+      script:
+        - mkdir build
+        - cd build
+        - CXX=clang++ CC=clang cmake -G Xcode -DRINGMESH_WITH_GRAPHICS:BOOL=OFF -DRINGMESH_WITH_UTILITIES:BOOL=ON -DRINGMESH_WITH_TESTS:BOOL=ON -DRINGMESH_WITH_TUTORIALS:BOOL=ON ..
+        - cmake --build . --config Debug -j2
+        - ctest -C Debug -j2
```

Comment:
Could you choose between ctest and cmake for running the tests and update the whole file with this convention?

Reply:
I do not understand. All the tests are running with ctest here.

Reply:
Except for coverage? Was it the one you thought?

Reply:
Ok, my mistake. I remembered wrong...